mdk/build.gradle

```diff
@@ -1,78 +1,166 @@
 buildscript {
     repositories {
-        maven { url = 'https://files.minecraftforge.net/maven' }
+        // Try to resolve locally first
+        mavenLocal()
```

Comment:
For the MDK example, this is bad. As its only for testing things. It'll cause more issues then it's worth for people who don't know what they are doing.